lms/views/lti/basic_launch.py

```diff
@@ -298,9 +298,11 @@ def _configure_js_to_show_document(self, document_url, assignment_gradable):
             # For students in Canvas with grades to submit we need to enable
```

Comment:
We already mention here "student" we are just making it explicit in the code (and fixing a bug in the process).